pkgs/applications/science/biology/psipred/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub, blast, tcsh, dbPath ? "uniref90.fasta", }:
+
+stdenv.mkDerivation rec {
+  pname = "psipred";
+  version = "4.0";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1frxpnrxkhj968k0w6r4npsn5g0jjd0bx6wmb68yqbqkns1qfrgf";
+  };
+
+  buildInputs = [ blast tcsh ];
+
+  postPatch = ''
+    substituteInPlace ./BLAST+/runpsipredplus \
+      --replace "uniref90filt" "${dbPath}" \
+      --replace "/usr/local/bin" "${blast}/bin" \
+      --replace "../bin" "$out/bin" \
+      --replace "../data" "$out/data"
```

Comment:
It should be an standard directory, `$out/share/psipred/data`, and ensure the installation puts the data in there.

Reply:
I fixed it, thanks!

Reply:
I fixed it, thanks!